src/browser_action/vitals.js

```diff
@@ -177,12 +177,6 @@
             overlayClosedForSession = true;
           });
           document.body.appendChild(overlayClose);
-        } else {
-          overlayClose.addEventListener('click', () => {
-            overlayElement.remove();
-            overlayClose.remove();
-            overlayClosedForSession = true;
-          });
```

Comment:
There's no point in adding multiple click handlers to close it. Only need it first time we create the button!